package.json

```diff
@@ -108,8 +108,8 @@
     "terser": "^3.10.1",
     "vue-eslint-parser": "^3.2.2",
     "vue-jest": "^3.0.0",
-    "vue-loader": "^15.4.2",
-    "vue-template-compiler": "^2.5.17"
+    "vue-loader": "^13.5.2",
```

Comment:
why downgrade? it will cause problems when used in main project

Reply:
it should work with 15. I didnt really used storyblock but I suppose you need to modify webpack config to add vue storyblock. Something like this

  config.module.rules.push({
    resourceQuery: /blockType=story/,
    loader: 'vue-storybook',
  })

Reply:
this whole stotyblcok is really dated and probably need to be updated to be properly used. Its from template and was never sued

Reply:
Okay i have resolved the issue but had to upgrade and add more Dev dependencies. this is outdated